python/ray/rllib/test/test_nested_spaces.py

```diff
@@ -340,6 +341,21 @@ def testMultiAgentComplexSpaces(self):
             self.assertEqual(seen[1][0].tolist(), cam_i)
             self.assertEqual(seen[2][0].tolist(), task_i)
 
+    def testRolloutDictSpace(self):
+        register_env("nested", lambda _: NestedDictEnv())
+        agent = PGAgent(env="nested")
+        agent.train()
+        path = agent.save()
+        agent.stop()
+
+        # Test train works on restore
+        agent2 = PGAgent(env="nested")
+        agent2.restore(path)
+        agent2.train()
+
+        # Test rollout works on restore
+        rollout(agent2, "nested", 100)
```

Comment:
wait this doesn't test that the preprocessing is handled correctly right?

Reply:
It crashes if it doesnt.